extensions/wikia/AffiliateService/js/units.js

```diff
@@ -1,22 +1,27 @@
 define('ext.wikia.AffiliateService.units', [], function () {
 	var units = [
 		{
-			category: 'disney2',
-			campaign: 'disneyplus2',
+			category: 'disneyplus',
+			campaign: 'mandalorian',
 			name: 'test unit2',
-			country: ['US', 'CA', 'NL', 'AU', 'NZ'],
-			header: 'Header 2',
-			subheader: 'This is subheader 2',
+			country: ['US'],
```

Comment:
Please bring back the proper GEO for both units.

Reply:
Gotcha!